lib/timers.js

```diff
@@ -389,12 +390,12 @@ const unenroll = exports.unenroll = function(item) {
 // Using existing objects as timers slightly reduces object overhead.
 exports.enroll = function(item, msecs) {
   if (typeof msecs !== 'number') {
-    throw new TypeError('"msecs" argument must be a number');
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'msecs',
+                               'number', msecs);
   }
 
   if (msecs < 0 || !isFinite(msecs)) {
-    throw new RangeError('"msecs" argument must be ' +
-                         'a non-negative finite number');
+    throw new errors.RangeError('ERR_INVALID_OPT_VALUE', 'msecs', msecs);
```

Comment:
The error message will become like:
```
RangeError [ERR_INVALID_ARG_TYPE]: The "msecs" argument must be of type a non-negative finite number
```
IMHO, we should add a new error code (e.g. `ERR_OPT_VALUE_OUT_OF_RANGE`), which can be reused in other modules. (e.g. https://github.com/nodejs/node/blob/master/lib/buffer.js#L244)